rune/libenclave/attestation/attester.go

```diff
@@ -38,3 +39,13 @@ func RegisterAttester(attester Attester) error {
 
 	return nil
 }
+
+func init() {
+	if err := RegisterAttester(&sgx_attester.SgxEcdsaAttester{}); err != nil {
+		fmt.Print(err)
+	}
+
+	if err := RegisterAttester(&sgx_attester.SgxEpidAttester{}); err != nil {
```

Comment:
OK all attestation specific logics need to explicitly call register function. So the register function doesn't need to be exported right?

Reply:
> OK all attestation specific logics need to explicitly call register function. So the register function doesn't need to be exported right?

ok，LGTM.